src/main/webapp/app/exercises/quiz/manage/re-evaluate/multiple-choice-question/re-evaluate-multiple-choice-question.component.html

```diff
@@ -1,132 +1,225 @@
+                            id="{{ 'cbRandomizeOrderMC' + questionIndex }}"
+                            [(ngModel)]="question.randomizeOrder"
+                            class="form-check-input custom-control-input"
+                        />
+                        <label
+                            class="form-check-label custom-control-label"
+                            for="{{ 'cbRandomizeOrderMC' + questionIndex }}"
+                            jhiTranslate="artemisApp.multipleChoiceQuestion.randomizeOrder"
+                        >
+                        </label>
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-lg-3 row justify-content-lg-end">
+                <div class="col-8 form-group">
                     <button
-                        class="btn btn-outline-secondary"
-                        [hidden]="isAnswerInvalid(answer)"
-                        (click)="setAnswerInvalid(answer)"
-                        jhiTranslate="artemisApp.quizExercise.re-evaluate.setAnswerInvalid"
+                        class="btn btn-outline-secondary question-invalid"
+                        [hidden]="question.invalid"
+                        (click)="question.invalid = true"
+                        jhiTranslate="artemisApp.quizExercise.re-evaluate.setQuestionInvalid"
                     ></button>
                     <button
-                        class="btn btn-outline-secondary"
-                        [hidden]="!isAnswerInvalid(answer)"
-                        [disabled]="isAnswerInvalid(answer)"
-                        jhiTranslate="artemisApp.quizExercise.re-evaluate.answerIsInvalid"
-                    ></button
-                    >&nbsp;
-                    <span class="re-evaluate-button" (click)="resetAnswer(answer)">
-                        <fa-icon [icon]="faUndo" size="2x"></fa-icon>
-                    </span>
-                    <span class="re-evaluate-button" (click)="deleteAnswer(answer)">
+                        class="btn btn-outline-secondary question-invalid"
+                        [hidden]="!question.invalid"
+                        [disabled]="question.invalid"
+                        jhiTranslate="artemisApp.quizExercise.re-evaluate.questionIsInvalid"
+                    ></button>
+                </div>
+                <div class="col-2">
+                    <div class="re-evaluate-button" id="delete-button" class="delete-button" (click)="delete()">
                         <fa-icon [icon]="faTrash" size="2x"></fa-icon>
-                    </span>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="markupEditorArea">
+            <div class="question-content" style="padding-bottom: 12px">
+                <div class="row">
+                    <div class="col-10 col-lg-11">
+                        <!-- TODO: Fix file upload    -->
```

Comment:
There is still a todo

Reply:
I'll create a follow-up PR for this since the error happens on other pages as well